seeds/populate_dummy_slider_images.js

```diff
@@ -0,0 +1,15 @@
+
+const seedSliderImages = require('../app/config/config').seedSliderImages;
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('slider').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('slider').insert([
+        {id: 1, image_url: seedSliderImages.image_url_1, created_at: new Date(), updated_at: new Date()},
```

Comment:
nice thinking, but lets not read these from config, they should be dry here.
a migration should represent the exact data and not dynamic otherwise problems will arise in the future where because say the config value changes which will then lead to unintended results in the DB et.c